### ISSUE TITLE
Is the web page being hosted?

### ISSUE THREAD

**Issue by sunrabbit123** [Ubinquitous/FTF_FindTheFollowers#1]
I was impressed with your project.
And I wanted to use it simply, but I don't think the web is being hosted.

If you are burdened with hosting, I recommend the following methods.

1. github pages
    this solution has `gh-pages` package and use case is very simple 
    insert script at package.json
    ```json
    "scripts": {
        "start": "react-scripts start",
        "build": "react-scripts build",
        "deploy": "gh-pages -d build"
      },
    ```
   
    and add workflow
    ```yml
    name: github pages
    
    on:
      push:
        branches:
          - main
    
    jobs:
      deploy:
        runs-on: ubuntu-18.04
        steps:
          - uses: actions/checkout@v2
    
          - name: Setup Node
            uses: actions/setup-node@v2.1.2
            with:
              node-version: "14.x"
    
          - run: yarn
          - run: yarn build
    
          - name: Deploy
            uses: peaceiris/actions-gh-pages@v3
            with:
              github_token: ${{ secrets.ACCESS_TOKEN }}
              publish_dir: ./build
    ```

    The solution will work beautifully.
2. railway
3. vercel
4. aws cloud front

The following four services are available free of charge or at reasonable prices.

thank you for reading it.

**Comment by sunrabbit123** [comment]
Of course, I can clone it, but I don't think others need to clone, I suggested it, so I hope you don't feel pressured.

**Comment by Ubinquitous** [comment]
Thanks for your interest in my project! When developing this project, I used the GitHub API for the first time, but I remember having a hard time because it didn't go my way.
Thanks for the trouble! As you said, I tried deploying using vercel. The address is https://ftf-find-the-followers.vercel.app! is very perfect but I don't like the long url. Haha.. If there is a chance to meet at the next concert, etc., I really want to meet you. thank you!
PS. I refer to the GRIG project a lot when studying!!
